### DISCUSSION TITLE
Supabse realtime with Bring your own database and RLS?

### DISCUSSION THREAD

**Discussion by Aadim1** [supabase/supabase#19239] [Questions]
Hello, I was wondering if it is possible to use Supabase Realtime with our own database and enforce RLS within it? I like the RLS feature of Realtime, and want to use within the database that I have right now.

 If it helps I am using AWS RDS.

**Comment by burggraf** [comment]
At this time, you'll only be able to use Supabase Realtime in the context of a Supabase project.  However, you could import your data into your Supabase project, or you might try using a FDW from inside your Supabase project to operate on your (external) database (without having to import the data.)

**Reply by johndpope** [reply]
i pulled apart the realtime / auth containers  added some debug stuff - 
https://github.com/supabase/supabase/issues/27153

I believe my forked containers now correctly boot up with external rds 

currently investigating the tenant database and whether it can work with the same db. 
at worst - it should be possible to wire up to a second rds instance.
i have a test_realtime script in my rds-supabase repo 
https://github.com/johndpope/rds-supabase/tree/main/docker
